### PR TITLE
Add .env support

### PR DESCRIPTION
**Pull request by drewvolz**
* This PR pulls the dotenv module from @hawkrives' [@degreepath/auditor](https://github.com/degreepath/auditor)
* Parses a `.env` when the main script for memberboat runs
* Provides a `.sample.env` file
* Updates documentation for running with python modules

### CODE REVIEW

File: memberboat/dotenv/dotenv.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+from typing import Union, Dict
+from pathlib import Path
+import os
+
+__author__ = "Hawken Rives"
+__copyright__ = "Copyright (C) 2019  Hawken MacKay Rives"
+__license__ = "AGPL"
+__version__ = "3.0"
+
```
[the remark this answers is not in the document]
Comment (drewvolz) [reply memberboat/dotenv/dotenv.py:9]: What did you have in mind? I was going off of [this](https://stackoverflow.com/a/1523456).

File: memberboat/dotenv/__init__.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from .dotenv import load
+
+__author__ = "Hawken Rives"
+__copyright__ = "Copyright (C) 2019  Hawken MacKay Rives"
+__license__ = "AGPL"
+__version__ = "3.0"
```
[the remark this answers is not in the document]
Comment (drewvolz) [reply memberboat/dotenv/__init__.py:6]: See comment above

File: memberboat/dotenv/dotenv.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (hawkrives) [reply memberboat/dotenv/dotenv.py:9]: I'd never seen that standard before. 

I'd be OK with a comment block of license text and a link to DP? Idk

[the remark this answers is not in the document]
Comment (drewvolz) [reply memberboat/dotenv/dotenv.py:9]: So the [original commit](https://github.com/rye/memberboat/pull/12/commits/bf704247f24eda4f6e37f83f0695bb262a127f02) is probably your preference @hawkrives?

**Pull request merged**